src/components/Search.css

```diff
@@ -0,0 +1,10 @@
+input{
+    margin: 10px;
```

Comment:
a második a te formázásod ami itt szerepel, látszik hogy a margint felülírja a default margin
![2024-08-01_13h57_37](https://github.com/user-attachments/assets/d0cf65c3-4cd3-4d58-a2b8-b1c2a47a2f8f)

3 opciód van
1. vagy egy specifikusabb selectort adsz ennek az elemnek, aminek magasabb a specificity-je (class, id, child selector, stb)
2. melléteszed hogy !important, ezt nem ajánlom, később nagyon nehéz lesz felülírni, nagyon ritkán használd ezt
3. reviewban lesz egy screenshot, oda írtam is hogy a form legyen flex, és használj gapet